spotifycli

```diff
@@ -75,8 +75,22 @@ def perform_spotify_action(spotify_command):
     Popen('dbus-send --print-reply --dest=org.mpris.MediaPlayer2.spotify ' \
     '/org/mpris/MediaPlayer2 org.mpris.MediaPlayer2.Player."%s"' % spotify_command, shell=True, stdout=PIPE)
 
+def get_sink_input():
+    if is_spotify_available():
+        curInput=-1
```

Comment:
could you please stick to snake_case naming convention? I'd prefer to change it to e.g. `current_input`